tests/analysis_test.py

```diff
@@ -60,3 +65,16 @@ def get_cmd_output(self, exec_name, options=''):
                              options, self.trace_writer.trace_root)
 
         return subprocess.getoutput(cmd)
+
+    def get_output(self, name, output):
+        out = open(os.path.join(self.trace_writer.trace_root, name), "w")
+        out.write(output)
+        out.close()
+        self.rm_trace = False
+
+    def diff(self, name, result, expected):
```

Comment:
Once again I think another more precise name might be beneficial. At least something that implies that this method's main purpose is to perform an assertion.